### PR TITLE
Add syncAt to settings so its easier to reset. More info in instructions

### PR DESCRIPTION
**Pull request by jacksonh**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -33,6 +33,14 @@ const settings: SettingSchemaDesc[] = [
...
     default: 60,
   },
+  {
+    key: 'synced at',
+    type: 'string',
+    title: 'Last Sync',
+    description:
+      'The last time Omnivore was synced. Clear this value to completely refresh the sync.',
+    default: '',
+  },
```
Comment (sywhb) [inline index.ts:43]: We might need to add the validation of the datetime string

Comment (jacksonh) [reply index.ts:43]: Good point, i guess if invalid we can just set to an empty string.

**Comment by sywhb** [comment]
we should probably close this one as i have merged mine @jacksonh

**Comment by jacksonh** [comment]
👍

**Pull request closed without merging**